rum-demo-app-ios/RUM Mobile Demo App/Extensions/UIViewControllerExtension.swift

```diff
@@ -0,0 +1,588 @@
+            (UIApplication.shared.delegate as? AppDelegate)?.changeRootViewController(loginNavController)
+        }
+    }
+    @objc func gotoPreviousScreen() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    func showMenuOption() {
+        
+        DispatchQueue.main.async {
+            let viewMainContainer = UIView.init(frame: window?.bounds ?? .zero)
+            viewMainContainer.backgroundColor = UIColor.black.withAlphaComponent(0.3)
+            let viewForMenu = UIView()
+            let lblTitle = UILabel.init(frame: CGRect.init(x: 0, y: 0, width: 150, height: 45))
+            lblTitle.text = "Change Config URL"
+            lblTitle.textAlignment = .center
+            lblTitle.textColor = .black
+            lblTitle.font = UIFont.systemFont(ofSize: 14)
+            lblTitle.backgroundColor = .white
+            viewForMenu.addSubview(lblTitle)
+            viewForMenu.frame.origin.x = UIScreen.main.bounds.width - (lblTitle.frame.width + 20)
+            viewForMenu.frame.origin.y = (window?.safeAreaInsets.top ?? 0) + 15
+            viewForMenu.frame.size.height = 45
+            viewForMenu.backgroundColor = .white
+            
+            
+            
+    //        let tapGesture = UITapGestureRecognizer.init(target: self, action: #selector(changeConfigURLButtonClicked(_:)))
+    //        tapGesture.delegate = self
+    //        viewForMenu.addGestureRecognizer(tapGesture)
+    //
+    //        let tapGestureOfMainView = UITapGestureRecognizer.init(target: self, action: #selector(tappedOutsideView(_:)))
+    //        tapGestureOfMainView.delegate = self
+    //        viewMainContainer.addGestureRecognizer(tapGestureOfMainView)
+            
+            let btnForDismiss = UIButton.init(frame: viewMainContainer.frame)
+            viewMainContainer.addSubview(btnForDismiss)
+            btnForDismiss.addTarget(self, action: #selector(self.tappedOutsideView(_:)), for: .touchUpInside)
+            btnForDismiss.tag = 112010
+            btnForDismiss.isUserInteractionEnabled = true
+            
+            viewMainContainer.addSubview(viewForMenu)
+            
+            let btn = UIButton.init(frame: viewForMenu.frame)
+            viewMainContainer.addSubview(btn)
+            btn.backgroundColor = UIColor.red.withAlphaComponent(0.5)
+            btn.addTarget(self, action: #selector(self.changeConfigURLButtonClicked(_:)), for: .touchUpInside)
+            btn.isUserInteractionEnabled = true
+            btn.isAccessibilityElement = true
+            btn.accessibilityLabel = "header_context_menu_item"
+            
+            viewMainContainer.tag = 102010
+    //        viewMainContainer.bringSubviewToFront(viewForMenu)
+            
+            window?.addSubview(viewMainContainer)
+        }
+    }
+    
+    @objc func changeConfigURLButtonClicked(_ sender : UIButton) {
+        print("CLICKED BUTTON ++ ==>")
```

Comment:
@akashp-splunk Can we remove this print statements?

Reply:
Done